nixos/doc/manual/release-notes/rl-2205.section.md

```diff
@@ -193,6 +193,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 - `security.klogd` was removed.  Logging of kernel messages is handled
   by systemd since Linux 3.5.
 
+- String 'configureFlags' is deprecated and will be removed in release 23.05. Please use a list of strings.
+
```

Comment:
It's good to document the change, but it seems noisy to bother readers with changes that they are equally well informed about by the evaluation warnings.

A "safe to skip" section would be nice.

Reply:
removed them

did it with `git checkout @~3 -- ./nixos/doc/manual/release-notes/rl-2205.section.md` to both the md and xml and then commited and fix'd up the commit. someone may find it useful so im posting it